Source/WebCore/rendering/svg/SVGRenderTreeAsText.cpp

```diff
@@ -488,7 +489,7 @@ void writeSVGResourceContainer(TextStream& ts, const RenderSVGResourceContainer&
         writeNameValuePair(ts, "markerUnits", marker.markerUnits());
         ts << " [ref at " << marker.referencePoint() << "]";
         ts << " [angle=";
-        if (marker.angle() == -1)
+        if (marker.markerElement().orientType() != SVGMarkerOrientAngle)
             ts << "auto" << "]\n";
         else
             ts << marker.angle() << "]\n";
```

Comment:
This also can be simplified like this:
```
    if (auto angle = this->angle())
        ts << *angle << "]\n";
    else
        ts << "auto" << "]\n";
```

Reply:
@shallawa - this change is leading to build failure. I fixed others and also tried to fix this twice as per my knowledge: This is build error:
```SVGRenderTreeAsText.cpp:492:26: error: invalid use of 'this' outside of a non-static member function```

Reply:
I have reverted to original, I tried to fix it but it seems to be not working.

Reply:
Sorry it is my mistake. I did not realize this change is in a different source file. I think this should work:
```
    if (auto angle = marker.angle())
        ts << *angle << "]\n";
    else
        ts << "auto" << "]\n";
```

Reply:
Pushed new build with suggested change. :-) Thanks for your help as always. @shallawa - one less bug in WebKit now.